databricks-dbutils-scala/src/main/scala/com/databricks/sdk/scala/dbutils/SdkDBUtilsImpl.scala

```diff
@@ -152,3 +162,54 @@ private class SdkSecretsUtils(client: WorkspaceClient) extends SecretUtils with
       SecretScope(scope.getName)
     }
 }
+
+private class SdkJobsUtils extends JobsUtils with NoHelp {
+  override val taskValues: TaskValuesUtils = new SdkTaskValues
+}
+
+private class SdkTaskValues extends TaskValuesUtils with NoHelp {
+  private var commandContext =
+    CommandContext()
```

Comment:
Do we need an instance here? It seems our integration does not rely on it. Can this be `private var commandContext: CommandContext = _`?

Reply:
I check that initially I return empty context